wdio/pages/applications.js

```diff
@@ -56,14 +58,23 @@ class Application extends SalesForce {
 
     collateralInfoField(fieldName) { return $(`//div[text() = '${fieldName}']/parent::div/following-sibling::div//input`) }
 
+
+    approvalConditionCheckBox(index) { return $(`(//span[text() = 'Approval Conditions']/../following-sibling::div//td[not(@style)])[${index}]`) }
+
+    saveNewButton(count=1) { return $(`(//div[@title='Save new' or @title='Save'])[${count}]`) }
+    addNewQuestionButtoninSC(count=1) { return $(`(//th[@class="actioncolumn"]//i)[${count}]`) }
+
     async searchApplicationWithId(applicationId) {
         await this.getElementWithAttribute('placeholder', 'Search this list...', 'input').setValue(`${applicationId}\n`);
     }
 
     async goToApplicationTabWithText(tabName) {
         let tabNameSelector = await $(`//div[text() = '${tabName}']//ancestor::li`);
-        await tabNameSelector.waitForDisplayed();
+        // await tabNameSelector.waitForExist();
+        await this.reloadIfElementNotPresent(tabNameSelector);
+        await tabNameSelector.waitForClickable();
```

Comment:
Due to the tabs not loading sometimes without refresh, use reloadIfNot{Displayed/Clickable} method.